Backend/src/main/resources/import.sql

```diff
@@ -1,9 +1,17 @@
 
 INSERT INTO color_type (color, available, created_at, updated_at) VALUES 
-('blue', true, current_timestamp, current_timestamp),
-('green', false, current_timestamp, current_timestamp),
-('purple', true, current_timestamp, current_timestamp), 
-('red', true, current_timestamp, current_timestamp);
+('#FFFFFF', true, current_timestamp, current_timestamp),
```

Comment:
I think we will want to stick to words and not hex codes because otherwise it could get crowded on the backend + messy for the user on the frontend. Thoughts?

Reply:
We'll have to use hex codes for the color-picker, but I thought of that problem & it's not an issue; when they select a color to add, it'll be a set list that we control, so the shades don't get out of control